### PR TITLE
Run laptop script in travis

### PR DESCRIPTION
**Pull request by asimpson**

### CODE REVIEW

File: mac [hunk 1]
```diff
@@ -44,13 +44,17 @@ fi
...
-case "$SHELL" in
-  */zsh) : ;;
-  *)
-    fancy_echo "Changing your shell to zsh ..."
-      chsh -s "$(which zsh)"
-    ;;
-esac
+# don't do this in CI
+if [[ -z "$TRAVIS" ]]; then
+  echo "Not in Travis..."
```
Comment (robtarr) [inline mac]: Do we want to remove this echo?

Comment (robtarr) [inline mac]: Does this fail in Travis?

File: mac [hunk 2]
```diff
@@ -162,8 +166,11 @@ for version in $nodeversions; do
 done
 
 # Link the last, most recent version
-brew unlink $mostrecentnode
-brew link --force $mostrecentnode
+# don't do this in CI
+if [[ -z "$TRAVIS" ]]; then
+  brew unlink $mostrecentnode
```
Comment (robtarr) [inline mac]: Does this fail in Travis?

File: mac (hunk 1 above)
Comment (asimpson) [reply mac]: yep

File: mac (hunk 2 above)
Comment (asimpson) [reply mac]: yep

**Pull request closed without merging**